ibm/mas_devops/playbooks/cp4d/db2wh-backup-restore.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Can you move the vars / defaults into the role (e.g. see example here https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/cp4d_db2wh/defaults/main.yml)

Reply:
Fixed.